wodles/azure/tests/test_azure.py

```diff
@@ -708,6 +709,47 @@ def test_get_blobs(mock_send, mock_update, blob_date, min_date, max_date, desire
             [blob.name for blob in blob_list if extension in blob.name])
 
 
+@patch('azure-logs.update_row_object')
+@patch('azure-logs.send_message')
+def test_get_blobs_only_with_prefix(mock_send, mock_update):
+    """Test get_blobs obtains the blobs only with prefix from a container and send their content to the socket."""
+    azure.args = MagicMock(
+        blobs=None, json_file=False, json_inline=False, reparse=False
+    )
+
+    prefix = "test_prefix"
+    blob_date_str = parse(FUTURE_DATE)
+
+    blob_list = [create_mocked_blob(blob_name=f"blob_{i}", last_modified=blob_date_str) for i in range(5)] + [
+        create_mocked_blob(blob_name=f"{prefix}/blob_{i}", last_modified=blob_date_str) for i in range(5)
+    ]
```

Comment:
Consider adding an additional set of files in a different prefix just to make sure these logs are being skipped in the same way logs placed in the root folder are.
```suggestion
    blob_list = [create_mocked_blob(blob_name=f"blob_{i}", last_modified=blob_date_str) for i in range(5)] + \
                [create_mocked_blob(blob_name=f"{prefix}/blob_{i}", last_modified=blob_date_str) for i in range(5) + \
                [create_mocked_blob(blob_name=f"other_prefix/blob_{i}", last_modified=blob_date_str) for i in range(5)]
```